discord_py_self_mcp/tools/permissions.py

```
@@ -0,0 +1,455 @@
+        target = guild.get_role(int(target_id))
+    elif target_type == "member":
+        target = guild.get_member(int(target_id))
+    else:
+        return None, f"Invalid target_type '{target_type}' — use 'role' or 'member'"
+
+    if not target:
+        return None, f"{target_type.capitalize()} {target_id} not found"
+    return target, None
+
+
+def _build_overwrite(arguments: dict):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_build_overwrite`

(ANN202)